utils/build-and-test

```diff
@@ -15,3 +15,10 @@ mkdir -p "${BUILDROOT}"
 cd "${BUILDROOT}"
 cmake -G Ninja -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ "${SRCROOT}"
 ninja test
+
+# run Swift package tests if `swift` is available
+if [ -x "$(command -v swift)" ]; then
+  swift test
+elif
+  echo 'Skipping Swift package tests because `swift` couldn\'t be found.'
+fi
```

Comment:
Note that this script is not part of Swift-CI either. This exists to facilitate local, Docker-based testing of llbuild.

Reply:
Ok, so it makes most sense to have it integrated in `lit` like @ddunbar wrote. Sounds like it still makes sense to keep that here?

Reply:
Well, at least until integrated with the lit tests.  It will be redundant then, and would be good to avoid the double build.